aip/0131.md

```diff
@@ -62,10 +63,18 @@ message GetBookRequest {
 
 - A resource name field **must** be included. It **should** be called `name`.
 - The comment for the `name` field **should** document the resource pattern.
+- The request message **must not** contain any other required fields, and
+  **should not** contain other optional fields unless they are described in
```

Comment:
...other option fields except those described in...

Reply:
Done (in all five).